frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/admission/CaseFileAdmission.js

```diff
@@ -522,6 +523,52 @@ function CaseFileAdmission({ t, path }) {
     });
   };
 
+  const handleDelayCondonation = async (caseDetails) => {
+    const documents = caseDetails?.caseDetails?.delayApplications?.formdata?.[0]?.data?.condonationFileUpload?.document
```

Comment:
I'm not sure if the case Schemas are updated or not, but try not to pick details from addtional details and formdata since we will be removing it after some time. If you can't pick it from the scemas, then only use addtionalDetails.formdata